schemaless/sources.py

```diff
@@ -709,7 +707,7 @@ class PermitAddendaSummary(Source):
         'earliest_addenda_arrival'
     ]
     DATA_SF = "https://data.sfgov.org/Housing-and-Buildings/Department-of-Building-Inspection-Permit-Addenda-w/87xy-gk8d"  # NOQA
-    DATA_SF_DOWNLOAD = "https://data.sfgov.org/api/views/87xy-gk8d/rows.csv?accessType=DOWNLOAD"  # NOQA
+    DATA_SF_VIEW_ID = "87xy-gk8d"
```

Comment:
Nit: Assuming we need to keep both DATA_SF and DATA_SF_VIEW_ID could we use the view id in the other one? So that if view id ever changes you don't have to update two places?

Reply:
I don't think it'd work. The DATA_SF link needs more than just the view_id -- note this one also has "Housing-and-Buildings" and "Department-of-Building-Inspection-Permit-Addenda-w".

Reply:
Isn't the view id part always the end of DATA_SF? My python isn't very good but would DATA_SF=https://data.sfgoov.org/foo/bar/" + DATA_SF_VIEW_ID? Regardless this is not a big issue at all, so approving